src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -123,6 +123,10 @@
   <data name="ContentTypeHeaderRequired" xml:space="preserve">
     <value>The "content-type" header is required.</value>
   </data>
+  <data name="CustomAuditHeaderTooLarge" xml:space="preserve">
+    <value>The maximum length of a custom audit header value is {0}. The supplied custom audit header '{1}' has length of {2}.</value>
+    <comment>PH0 is a constant defining the max length of a header.  PH1 is the name  of the header sent in the request.  PH2 is the length of the incoming header value.  Example: The maximum length of a custom audit header value is 2048. The supplied custom audit header 'X-MS-AZUREFHIR-AUDIT-SITE' has length of 3072.</comment>
```

Comment:
In other comments we've used `{0}` instead of `PH0`

Reply:
This is actually correct @brendankowitz - Using curly braces { } in a comment will get interpreted by the Localization engine as a rule to be interpreted e.g. {Locked} {NumberedPlaceholder="{0}"} etc. So to refer to curly braces but not as rules, we cannot use { } and the convention set is to refer to them as PH0, PH1 etc.  If other places have used { } in comments that are not rules, they should change to PH0...

Reply:
Thanks for clarification @JoeDRowan, just calling this out because the file is not consistent